src/status_im/translations/ja.cljs

```diff
@@ -2,36 +2,39 @@
 
 (def translations
   {
-   ;common
-   :members-title                         "メンバー"
-   :not-implemented                       "実装されていません"
-   :chat-name                             "チャット名"
-   :notifications-title                   "通知と音声"
+   ;;common
+   :members-title                       　　　　"メンバー"
```

Comment:
Why did you add these spaces? Shouldn't be an issue, but please delete so all content lines up for better readability